src/main/java/com/puppycrawl/tools/checkstyle/filters/PatchXpathFilterElement.java

```diff
@@ -263,6 +284,13 @@ private DetailAST getEventAst(TreeWalkerAuditEvent event) {
         return eventAst;
     }
 
+    /**
+     * Get event's ast node's all child nodes' max and min lineNo
+     * using iterative algorithm.
+     *
+     * @param ast DetailAST event's corresponding ast node
+     * @return Map contains ast node's all child nodes' max and min lineNo
```

Comment:
`event's` - this method doesn't know anything about an event, it works directly with AST node.

```
Get event's ast node's all child nodes' max and min lineNo
using iterative algorithm.
```
-->
`Find min and max line numbers from AST node and its children`